diff --git a/Common/sources/commondefines.js b/Common/sources/commondefines.js
index 4518ac8..34ab4e2 100644
--- a/Common/sources/commondefines.js
+++ b/Common/sources/commondefines.js
@@ -970,8 +970,8 @@ const c_oAscQueueType = {
   activemq: 'activemq'
 };
 
-const buildVersion = '4.1.2';
-const buildNumber = 37;
+const buildVersion = '5.5.1';
+const buildNumber = 76;
 
 exports.TaskQueueData = TaskQueueData;
 exports.CMailMergeSendData = CMailMergeSendData;
diff --git a/Common/sources/license.js b/Common/sources/license.js
index 290d85d..c136956 100644
--- a/Common/sources/license.js
+++ b/Common/sources/license.js
@@ -32,53 +32,26 @@
 
 'use strict';
 
-const config = require('config');
-const configL = config.get('license');
 const constants = require('./constants');
-const logger = require('./logger');
-const editorDataStorage = require('./../../DocService/sources/' + config.get('services.CoAuthoring.server.editorDataStorage'));
-
 const buildDate = '6/29/2016';
 const oBuildDate = new Date(buildDate);
-const oPackageType = configL.get('packageType');
-
-const cfgRedisPrefix = config.get('services.CoAuthoring.redis.prefix');
-const redisKeyLicense = cfgRedisPrefix + constants.REDIS_KEY_LICENSE;
-
-let editorData = new editorDataStorage();
 
 exports.readLicense = function*() {
-	const c_LR = constants.LICENSE_RESULT;
-	const res = {
+	return {
 		count: 1,
-		type: c_LR.Error,
+		type: constants.LICENSE_RESULT.Success,
 		light: false,
-		packageType: oPackageType,
+		packageType: constants.PACKAGE_TYPE_I,
 		mode: constants.LICENSE_MODE.None,
 		branding: false,
-		connections: constants.LICENSE_CONNECTIONS,
+		connections: 9999,
 		customization: false,
-		usersCount: 0,
-		usersExpire: constants.LICENSE_EXPIRE_USERS_ONE_DAY,
-		hasLicense: false,
+		usersCount: 9999,
+		usersExpire: 99999,
+		hasLicense: true,
 		plugins: false,
 		buildDate: oBuildDate,
-		endDate: null
+		endDate: "2099-01-01T23:59:59.000Z"
 	};
-
-	if (yield* _getFileState()) {
-		res.type = c_LR.ExpiredTrial;
-	}
-
-	if (res.type === c_LR.Expired || res.type === c_LR.ExpiredTrial) {
-		res.count = 1;
-		logger.error('License: License Expired!!!');
-	}
-
-	return res;
 };
-exports.packageType = oPackageType;
-
-function* _getFileState() {
-	return yield editorData.getLicense(redisKeyLicense);
-}
+exports.packageType = constants.PACKAGE_TYPE_I;
diff --git a/Makefile b/Makefile
index 40bf93e..7109dbd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-GRUNT = grunt
+GRUNT = ./node_modules/.bin/grunt
 GRUNT_FLAGS = --no-color -v
 
 GRUNT_FILES = Gruntfile.js.out
@@ -46,7 +46,7 @@ endif
 
 TARGET := $(PLATFORM)_$(ARCHITECTURE)
 
-OUTPUT = ../build_tools/out/$(TARGET)/onlyoffice/documentserver/server
+OUTPUT = build/out/$(TARGET)/onlyoffice/documentserver/server
 
 SPELLCHECKER_DICTIONARIES := $(OUTPUT)/SpellChecker/dictionaries
 SPELLCHECKER_DICTIONARY_FILES += ../dictionaries/*_*
@@ -83,7 +83,7 @@ DEBUG = $(BRANDING_DIR)/debug.js
 .PHONY: all clean install uninstall build-date
 
 .NOTPARALLEL:
-all: $(SPELLCHECKER_DICTIONARIES) $(TOOLS) $(SCHEMA) $(CORE_FONTS) $(LICENSE) $(WELCOME) $(INFO) build-date
+all: build-date
 
 build-date: $(GRUNT_FILES)
 	sed "s|\(const buildVersion = \).*|\1'${PRODUCT_VERSION}';|" -i $(COMMON_DEFINES_JS)
